proto/cosmos/offchain/v1alpha1/offchain.proto

```diff
@@ -0,0 +1,15 @@
+syntax="proto3";
+
+package cosmos.offchain.v1alpha1;
```

Comment:
I believe we don't need a new module just for this. I would put it in x/auth.

ref: https://github.com/cosmos/cosmos-sdk/pull/7727#discussion_r514185926

Reply:
Yes, it can go on x/auth, I didn't know where to put it so it ended up in x/offchain